### ISSUE TITLE
Lambert Conformal example errors due to coastlines dependency

### ISSUE THREAD

**Issue by tom-spalding** [SciTools/cartopy#526]
The coastline dependency `http://www.nacis.org/naturalearth/110m/physical/ne_110m_coastline.zip` 404's.

```
#%pylab inline
# Cartopy Lambert Conformal test 
# http://scitools.org.uk/cartopy/docs/latest/crs/projections.html#lambertconformal

import matplotlib.pyplot as plt
import cartopy.crs as ccrs

plt.figure(figsize=(4.28969332205, 3))
ax = plt.axes(projection=ccrs.LambertConformal())
ax.coastlines(resolution='110m')
ax.gridlines()

plt.show()
```

If I restart the kernel and comment out `%pylab inline`.

```
/usr/local/lib/python2.7/dist-packages/cartopy/io/__init__.py:261: DownloadWarning: Downloading: http://www.nacis.org/naturalearth/110m/physical/ne_110m_coastline.zip
  warnings.warn('Downloading: {}'.format(url), DownloadWarning)
Exception in Tkinter callback
Traceback (most recent call last):
  File "/usr/lib/python2.7/lib-tk/Tkinter.py", line 1489, in __call__
    return self.func(*args)
  File "/usr/lib/pymodules/python2.7/matplotlib/backends/backend_tkagg.py", line 276, in resize
    self.show()
  File "/usr/lib/pymodules/python2.7/matplotlib/backends/backend_tkagg.py", line 348, in draw
    FigureCanvasAgg.draw(self)
  File "/usr/lib/pymodules/python2.7/matplotlib/backends/backend_agg.py", line 451, in draw
    self.figure.draw(self.renderer)
  File "/usr/lib/pymodules/python2.7/matplotlib/artist.py", line 55, in draw_wrapper
    draw(artist, renderer, *args, **kwargs)
  File "/usr/lib/pymodules/python2.7/matplotlib/figure.py", line 1034, in draw
    func(*args)
  File "/usr/lib/pymodules/python2.7/matplotlib/artist.py", line 55, in draw_wrapper
    draw(artist, renderer, *args, **kwargs)
  File "/usr/local/lib/python2.7/dist-packages/cartopy/mpl/geoaxes.py", line 355, in draw
    inframe=inframe)
  File "/usr/lib/pymodules/python2.7/matplotlib/artist.py", line 55, in draw_wrapper
    draw(artist, renderer, *args, **kwargs)
  File "/usr/lib/pymodules/python2.7/matplotlib/axes.py", line 2086, in draw
    a.draw(renderer)
  File "/usr/lib/pymodules/python2.7/matplotlib/artist.py", line 55, in draw_wrapper
    draw(artist, renderer, *args, **kwargs)
  File "/usr/local/lib/python2.7/dist-packages/cartopy/mpl/feature_artist.py", line 103, in draw
    geoms = self._feature.intersecting_geometries(extent)
  File "/usr/local/lib/python2.7/dist-packages/cartopy/feature.py", line 113, in intersecting_geometries
    return (geom for geom in self.geometries() if
  File "/usr/local/lib/python2.7/dist-packages/cartopy/feature.py", line 180, in geometries
    name=self.name)
  File "/usr/local/lib/python2.7/dist-packages/cartopy/io/shapereader.py", line 246, in natural_earth
    return ne_downloader.path(format_dict)
  File "/usr/local/lib/python2.7/dist-packages/cartopy/io/__init__.py", line 223, in path
    result_path = self.acquire_resource(target_path, format_dict)
  File "/usr/local/lib/python2.7/dist-packages/cartopy/io/shapereader.py", line 301, in acquire_resource
    shapefile_online = self._urlopen(url)
  File "/usr/local/lib/python2.7/dist-packages/cartopy/io/__init__.py", line 262, in _urlopen
    return urlopen(url)
  File "/usr/lib/python2.7/urllib2.py", line 127, in urlopen
    return _opener.open(url, data, timeout)
  File "/usr/lib/python2.7/urllib2.py", line 410, in open
    response = meth(req, response)
  File "/usr/lib/python2.7/urllib2.py", line 523, in http_response
    'http', request, response, code, msg, hdrs)
  File "/usr/lib/python2.7/urllib2.py", line 442, in error
    result = self._call_chain(*args)
  File "/usr/lib/python2.7/urllib2.py", line 382, in _call_chain
    result = func(*args)
  File "/usr/lib/python2.7/urllib2.py", line 629, in http_error_302
    return self.parent.open(new, timeout=req.timeout)
  File "/usr/lib/python2.7/urllib2.py", line 410, in open
    response = meth(req, response)
  File "/usr/lib/python2.7/urllib2.py", line 523, in http_response
    'http', request, response, code, msg, hdrs)
  File "/usr/lib/python2.7/urllib2.py", line 448, in error
    return self._call_chain(*args)
  File "/usr/lib/python2.7/urllib2.py", line 382, in _call_chain
    result = func(*args)
  File "/usr/lib/python2.7/urllib2.py", line 531, in http_error_default
    raise HTTPError(req.get_full_url(), code, msg, hdrs, fp)
HTTPError: HTTP Error 404: Not Found
Exception in Tkinter callback
Traceback (most recent call last):
  File "/usr/lib/python2.7/lib-tk/Tkinter.py", line 1489, in __call__
    return self.func(*args)
  File "/usr/lib/pymodules/python2.7/matplotlib/backends/backend_tkagg.py", line 276, in resize
    self.show()
  File "/usr/lib/pymodules/python2.7/matplotlib/backends/backend_tkagg.py", line 348, in draw
    FigureCanvasAgg.draw(self)
  File "/usr/lib/pymodules/python2.7/matplotlib/backends/backend_agg.py", line 451, in draw
    self.figure.draw(self.renderer)
  File "/usr/lib/pymodules/python2.7/matplotlib/artist.py", line 55, in draw_wrapper
    draw(artist, renderer, *args, **kwargs)
  File "/usr/lib/pymodules/python2.7/matplotlib/figure.py", line 1034, in draw
    func(*args)
  File "/usr/lib/pymodules/python2.7/matplotlib/artist.py", line 55, in draw_wrapper
    draw(artist, renderer, *args, **kwargs)
  File "/usr/local/lib/python2.7/dist-packages/cartopy/mpl/geoaxes.py", line 355, in draw
    inframe=inframe)
  File "/usr/lib/pymodules/python2.7/matplotlib/artist.py", line 55, in draw_wrapper
    draw(artist, renderer, *args, **kwargs)
  File "/usr/lib/pymodules/python2.7/matplotlib/axes.py", line 2086, in draw
    a.draw(renderer)
  File "/usr/lib/pymodules/python2.7/matplotlib/artist.py", line 55, in draw_wrapper
    draw(artist, renderer, *args, **kwargs)
  File "/usr/local/lib/python2.7/dist-packages/cartopy/mpl/feature_artist.py", line 103, in draw
    geoms = self._feature.intersecting_geometries(extent)
  File "/usr/local/lib/python2.7/dist-packages/cartopy/feature.py", line 113, in intersecting_geometries
    return (geom for geom in self.geometries() if
  File "/usr/local/lib/python2.7/dist-packages/cartopy/feature.py", line 180, in geometries
    name=self.name)
  File "/usr/local/lib/python2.7/dist-packages/cartopy/io/shapereader.py", line 246, in natural_earth
    return ne_downloader.path(format_dict)
  File "/usr/local/lib/python2.7/dist-packages/cartopy/io/__init__.py", line 223, in path
    result_path = self.acquire_resource(target_path, format_dict)
  File "/usr/local/lib/python2.7/dist-packages/cartopy/io/shapereader.py", line 301, in acquire_resource
    shapefile_online = self._urlopen(url)
  File "/usr/local/lib/python2.7/dist-packages/cartopy/io/__init__.py", line 262, in _urlopen
    return urlopen(url)
  File "/usr/lib/python2.7/urllib2.py", line 127, in urlopen
    return _opener.open(url, data, timeout)
  File "/usr/lib/python2.7/urllib2.py", line 410, in open
    response = meth(req, response)
  File "/usr/lib/python2.7/urllib2.py", line 523, in http_response
    'http', request, response, code, msg, hdrs)
  File "/usr/lib/python2.7/urllib2.py", line 442, in error
    result = self._call_chain(*args)
  File "/usr/lib/python2.7/urllib2.py", line 382, in _call_chain
    result = func(*args)
  File "/usr/lib/python2.7/urllib2.py", line 629, in http_error_302
    return self.parent.open(new, timeout=req.timeout)
  File "/usr/lib/python2.7/urllib2.py", line 410, in open
    response = meth(req, response)
  File "/usr/lib/python2.7/urllib2.py", line 523, in http_response
    'http', request, response, code, msg, hdrs)
  File "/usr/lib/python2.7/urllib2.py", line 448, in error
    return self._call_chain(*args)
  File "/usr/lib/python2.7/urllib2.py", line 382, in _call_chain
    result = func(*args)
  File "/usr/lib/python2.7/urllib2.py", line 531, in http_error_default
    raise HTTPError(req.get_full_url(), code, msg, hdrs, fp)
HTTPError: HTTP Error 404: Not Found
Exception in Tkinter callback
Traceback (most recent call last):
  File "/usr/lib/python2.7/lib-tk/Tkinter.py", line 1489, in __call__
    return self.func(*args)
  File "/usr/lib/pymodules/python2.7/matplotlib/backends/backend_tkagg.py", line 276, in resize
    self.show()
  File "/usr/lib/pymodules/python2.7/matplotlib/backends/backend_tkagg.py", line 348, in draw
    FigureCanvasAgg.draw(self)
  File "/usr/lib/pymodules/python2.7/matplotlib/backends/backend_agg.py", line 451, in draw
    self.figure.draw(self.renderer)
  File "/usr/lib/pymodules/python2.7/matplotlib/artist.py", line 55, in draw_wrapper
    draw(artist, renderer, *args, **kwargs)
  File "/usr/lib/pymodules/python2.7/matplotlib/figure.py", line 1034, in draw
    func(*args)
  File "/usr/lib/pymodules/python2.7/matplotlib/artist.py", line 55, in draw_wrapper
    draw(artist, renderer, *args, **kwargs)
  File "/usr/local/lib/python2.7/dist-packages/cartopy/mpl/geoaxes.py", line 355, in draw
    inframe=inframe)
  File "/usr/lib/pymodules/python2.7/matplotlib/artist.py", line 55, in draw_wrapper
    draw(artist, renderer, *args, **kwargs)
  File "/usr/lib/pymodules/python2.7/matplotlib/axes.py", line 2086, in draw
    a.draw(renderer)
  File "/usr/lib/pymodules/python2.7/matplotlib/artist.py", line 55, in draw_wrapper
    draw(artist, renderer, *args, **kwargs)
  File "/usr/local/lib/python2.7/dist-packages/cartopy/mpl/feature_artist.py", line 103, in draw
    geoms = self._feature.intersecting_geometries(extent)
  File "/usr/local/lib/python2.7/dist-packages/cartopy/feature.py", line 113, in intersecting_geometries
    return (geom for geom in self.geometries() if
  File "/usr/local/lib/python2.7/dist-packages/cartopy/feature.py", line 180, in geometries
    name=self.name)
  File "/usr/local/lib/python2.7/dist-packages/cartopy/io/shapereader.py", line 246, in natural_earth
    return ne_downloader.path(format_dict)
  File "/usr/local/lib/python2.7/dist-packages/cartopy/io/__init__.py", line 223, in path
    result_path = self.acquire_resource(target_path, format_dict)
  File "/usr/local/lib/python2.7/dist-packages/cartopy/io/shapereader.py", line 301, in acquire_resource
    shapefile_online = self._urlopen(url)
  File "/usr/local/lib/python2.7/dist-packages/cartopy/io/__init__.py", line 262, in _urlopen
    return urlopen(url)
  File "/usr/lib/python2.7/urllib2.py", line 127, in urlopen
    return _opener.open(url, data, timeout)
  File "/usr/lib/python2.7/urllib2.py", line 410, in open
    response = meth(req, response)
  File "/usr/lib/python2.7/urllib2.py", line 523, in http_response
    'http', request, response, code, msg, hdrs)
  File "/usr/lib/python2.7/urllib2.py", line 442, in error
    result = self._call_chain(*args)
  File "/usr/lib/python2.7/urllib2.py", line 382, in _call_chain
    result = func(*args)
  File "/usr/lib/python2.7/urllib2.py", line 629, in http_error_302
    return self.parent.open(new, timeout=req.timeout)
  File "/usr/lib/python2.7/urllib2.py", line 410, in open
    response = meth(req, response)
  File "/usr/lib/python2.7/urllib2.py", line 523, in http_response
    'http', request, response, code, msg, hdrs)
  File "/usr/lib/python2.7/urllib2.py", line 448, in error
    return self._call_chain(*args)
  File "/usr/lib/python2.7/urllib2.py", line 382, in _call_chain
    result = func(*args)
  File "/usr/lib/python2.7/urllib2.py", line 531, in http_error_default
    raise HTTPError(req.get_full_url(), code, msg, hdrs, fp)
HTTPError: HTTP Error 404: Not Found
Exception in Tkinter callback
Traceback (most recent call last):
  File "/usr/lib/python2.7/lib-tk/Tkinter.py", line 1489, in __call__
    return self.func(*args)
  File "/usr/lib/pymodules/python2.7/matplotlib/backends/backend_tkagg.py", line 276, in resize
    self.show()
  File "/usr/lib/pymodules/python2.7/matplotlib/backends/backend_tkagg.py", line 348, in draw
    FigureCanvasAgg.draw(self)
  File "/usr/lib/pymodules/python2.7/matplotlib/backends/backend_agg.py", line 451, in draw
    self.figure.draw(self.renderer)
  File "/usr/lib/pymodules/python2.7/matplotlib/artist.py", line 55, in draw_wrapper
    draw(artist, renderer, *args, **kwargs)
  File "/usr/lib/pymodules/python2.7/matplotlib/figure.py", line 1034, in draw
    func(*args)
  File "/usr/lib/pymodules/python2.7/matplotlib/artist.py", line 55, in draw_wrapper
    draw(artist, renderer, *args, **kwargs)
  File "/usr/local/lib/python2.7/dist-packages/cartopy/mpl/geoaxes.py", line 355, in draw
    inframe=inframe)
  File "/usr/lib/pymodules/python2.7/matplotlib/artist.py", line 55, in draw_wrapper
    draw(artist, renderer, *args, **kwargs)
  File "/usr/lib/pymodules/python2.7/matplotlib/axes.py", line 2086, in draw
    a.draw(renderer)
  File "/usr/lib/pymodules/python2.7/matplotlib/artist.py", line 55, in draw_wrapper
    draw(artist, renderer, *args, **kwargs)
  File "/usr/local/lib/python2.7/dist-packages/cartopy/mpl/feature_artist.py", line 103, in draw
    geoms = self._feature.intersecting_geometries(extent)
  File "/usr/local/lib/python2.7/dist-packages/cartopy/feature.py", line 113, in intersecting_geometries
    return (geom for geom in self.geometries() if
  File "/usr/local/lib/python2.7/dist-packages/cartopy/feature.py", line 180, in geometries
    name=self.name)
  File "/usr/local/lib/python2.7/dist-packages/cartopy/io/shapereader.py", line 246, in natural_earth
    return ne_downloader.path(format_dict)
  File "/usr/local/lib/python2.7/dist-packages/cartopy/io/__init__.py", line 223, in path
    result_path = self.acquire_resource(target_path, format_dict)
  File "/usr/local/lib/python2.7/dist-packages/cartopy/io/shapereader.py", line 301, in acquire_resource
    shapefile_online = self._urlopen(url)
  File "/usr/local/lib/python2.7/dist-packages/cartopy/io/__init__.py", line 262, in _urlopen
    return urlopen(url)
  File "/usr/lib/python2.7/urllib2.py", line 127, in urlopen
    return _opener.open(url, data, timeout)
  File "/usr/lib/python2.7/urllib2.py", line 410, in open
    response = meth(req, response)
  File "/usr/lib/python2.7/urllib2.py", line 523, in http_response
    'http', request, response, code, msg, hdrs)
  File "/usr/lib/python2.7/urllib2.py", line 442, in error
    result = self._call_chain(*args)
  File "/usr/lib/python2.7/urllib2.py", line 382, in _call_chain
    result = func(*args)
  File "/usr/lib/python2.7/urllib2.py", line 629, in http_error_302
    return self.parent.open(new, timeout=req.timeout)
  File "/usr/lib/python2.7/urllib2.py", line 410, in open
    response = meth(req, response)
  File "/usr/lib/python2.7/urllib2.py", line 523, in http_response
    'http', request, response, code, msg, hdrs)
  File "/usr/lib/python2.7/urllib2.py", line 448, in error
    return self._call_chain(*args)
  File "/usr/lib/python2.7/urllib2.py", line 382, in _call_chain
    result = func(*args)
  File "/usr/lib/python2.7/urllib2.py", line 531, in http_error_default
    raise HTTPError(req.get_full_url(), code, msg, hdrs, fp)
HTTPError: HTTP Error 404: Not Found
```

**Comment by cpaulik** [comment]
Example still fails in release 0.11.2 from Github. The URL's are still wrong.

```
import matplotlib.pyplot as plt
import cartopy.crs as ccrs
import cartopy
print cartopy.__version__

plt.figure(figsize=(4.28969332205, 3))
ax = plt.axes(projection=ccrs.LambertConformal())
ax.coastlines(resolution='110m')
ax.gridlines()

plt.show()
```

```
/home/pydev/.virtualenvs/genpurp/local/lib/python2.7/site-packages/cartopy/io/__init__.py:261: DownloadWarning: Downloading: https://github.com/nvkelso/natural-earth-vector/raw/master/zips/110m_physical/ne_110m_coastline.zip
  warnings.warn('Downloading: {}'.format(url), DownloadWarning)
Exception in Tkinter callback
Traceback (most recent call last):
  File "/usr/lib/python2.7/lib-tk/Tkinter.py", line 1489, in __call__
    return self.func(*args)
  File "/home/pydev/.virtualenvs/genpurp/local/lib/python2.7/site-packages/matplotlib/backends/backend_tkagg.py", line 278, in resize
    self.show()
  File "/home/pydev/.virtualenvs/genpurp/local/lib/python2.7/site-packages/matplotlib/backends/backend_tkagg.py", line 349, in draw
    FigureCanvasAgg.draw(self)
  File "/home/pydev/.virtualenvs/genpurp/local/lib/python2.7/site-packages/matplotlib/backends/backend_agg.py", line 469, in draw
    self.figure.draw(self.renderer)
  File "/home/pydev/.virtualenvs/genpurp/local/lib/python2.7/site-packages/matplotlib/artist.py", line 59, in draw_wrapper
    draw(artist, renderer, *args, **kwargs)
  File "/home/pydev/.virtualenvs/genpurp/local/lib/python2.7/site-packages/matplotlib/figure.py", line 1079, in draw
    func(*args)
  File "/home/pydev/.virtualenvs/genpurp/local/lib/python2.7/site-packages/matplotlib/artist.py", line 59, in draw_wrapper
    draw(artist, renderer, *args, **kwargs)
  File "/home/pydev/.virtualenvs/genpurp/local/lib/python2.7/site-packages/cartopy/mpl/geoaxes.py", line 355, in draw
    inframe=inframe)
  File "/home/pydev/.virtualenvs/genpurp/local/lib/python2.7/site-packages/matplotlib/artist.py", line 59, in draw_wrapper
    draw(artist, renderer, *args, **kwargs)
  File "/home/pydev/.virtualenvs/genpurp/local/lib/python2.7/site-packages/matplotlib/axes/_base.py", line 2092, in draw
    a.draw(renderer)
  File "/home/pydev/.virtualenvs/genpurp/local/lib/python2.7/site-packages/matplotlib/artist.py", line 59, in draw_wrapper
    draw(artist, renderer, *args, **kwargs)
  File "/home/pydev/.virtualenvs/genpurp/local/lib/python2.7/site-packages/cartopy/mpl/feature_artist.py", line 103, in draw
    geoms = self._feature.intersecting_geometries(extent)
  File "/home/pydev/.virtualenvs/genpurp/local/lib/python2.7/site-packages/cartopy/feature.py", line 113, in intersecting_geometries
    return (geom for geom in self.geometries() if
  File "/home/pydev/.virtualenvs/genpurp/local/lib/python2.7/site-packages/cartopy/feature.py", line 180, in geometries
    name=self.name)
  File "/home/pydev/.virtualenvs/genpurp/local/lib/python2.7/site-packages/cartopy/io/shapereader.py", line 261, in natural_earth
    return ne_downloader.path(format_dict)
  File "/home/pydev/.virtualenvs/genpurp/local/lib/python2.7/site-packages/cartopy/io/__init__.py", line 223, in path
    result_path = self.acquire_resource(target_path, format_dict)
  File "/home/pydev/.virtualenvs/genpurp/local/lib/python2.7/site-packages/cartopy/io/shapereader.py", line 320, in acquire_resource
    shapefile_online = self._urlopen(url)
  File "/home/pydev/.virtualenvs/genpurp/local/lib/python2.7/site-packages/cartopy/io/__init__.py", line 262, in _urlopen
    return urlopen(url)
  File "/usr/lib/python2.7/urllib2.py", line 127, in urlopen
    return _opener.open(url, data, timeout)
  File "/usr/lib/python2.7/urllib2.py", line 410, in open
    response = meth(req, response)
  File "/usr/lib/python2.7/urllib2.py", line 523, in http_response
    'http', request, response, code, msg, hdrs)
  File "/usr/lib/python2.7/urllib2.py", line 448, in error
    return self._call_chain(*args)
  File "/usr/lib/python2.7/urllib2.py", line 382, in _call_chain
    result = func(*args)
  File "/usr/lib/python2.7/urllib2.py", line 531, in http_error_default
    raise HTTPError(req.get_full_url(), code, msg, hdrs, fp)
HTTPError: HTTP Error 404: Not Found
Exception in Tkinter callback
Traceback (most recent call last):
  File "/usr/lib/python2.7/lib-tk/Tkinter.py", line 1489, in __call__
    return self.func(*args)
  File "/home/pydev/.virtualenvs/genpurp/local/lib/python2.7/site-packages/matplotlib/backends/backend_tkagg.py", line 278, in resize
    self.show()
  File "/home/pydev/.virtualenvs/genpurp/local/lib/python2.7/site-packages/matplotlib/backends/backend_tkagg.py", line 349, in draw
    FigureCanvasAgg.draw(self)
  File "/home/pydev/.virtualenvs/genpurp/local/lib/python2.7/site-packages/matplotlib/backends/backend_agg.py", line 469, in draw
    self.figure.draw(self.renderer)
  File "/home/pydev/.virtualenvs/genpurp/local/lib/python2.7/site-packages/matplotlib/artist.py", line 59, in draw_wrapper
    draw(artist, renderer, *args, **kwargs)
  File "/home/pydev/.virtualenvs/genpurp/local/lib/python2.7/site-packages/matplotlib/figure.py", line 1079, in draw
    func(*args)
  File "/home/pydev/.virtualenvs/genpurp/local/lib/python2.7/site-packages/matplotlib/artist.py", line 59, in draw_wrapper
    draw(artist, renderer, *args, **kwargs)
  File "/home/pydev/.virtualenvs/genpurp/local/lib/python2.7/site-packages/cartopy/mpl/geoaxes.py", line 355, in draw
    inframe=inframe)
  File "/home/pydev/.virtualenvs/genpurp/local/lib/python2.7/site-packages/matplotlib/artist.py", line 59, in draw_wrapper
    draw(artist, renderer, *args, **kwargs)
  File "/home/pydev/.virtualenvs/genpurp/local/lib/python2.7/site-packages/matplotlib/axes/_base.py", line 2092, in draw
    a.draw(renderer)
  File "/home/pydev/.virtualenvs/genpurp/local/lib/python2.7/site-packages/matplotlib/artist.py", line 59, in draw_wrapper
    draw(artist, renderer, *args, **kwargs)
  File "/home/pydev/.virtualenvs/genpurp/local/lib/python2.7/site-packages/cartopy/mpl/feature_artist.py", line 103, in draw
    geoms = self._feature.intersecting_geometries(extent)
  File "/home/pydev/.virtualenvs/genpurp/local/lib/python2.7/site-packages/cartopy/feature.py", line 113, in intersecting_geometries
    return (geom for geom in self.geometries() if
  File "/home/pydev/.virtualenvs/genpurp/local/lib/python2.7/site-packages/cartopy/feature.py", line 180, in geometries
    name=self.name)
  File "/home/pydev/.virtualenvs/genpurp/local/lib/python2.7/site-packages/cartopy/io/shapereader.py", line 261, in natural_earth
    return ne_downloader.path(format_dict)
  File "/home/pydev/.virtualenvs/genpurp/local/lib/python2.7/site-packages/cartopy/io/__init__.py", line 223, in path
    result_path = self.acquire_resource(target_path, format_dict)
  File "/home/pydev/.virtualenvs/genpurp/local/lib/python2.7/site-packages/cartopy/io/shapereader.py", line 320, in acquire_resource
    shapefile_online = self._urlopen(url)
  File "/home/pydev/.virtualenvs/genpurp/local/lib/python2.7/site-packages/cartopy/io/__init__.py", line 262, in _urlopen
    return urlopen(url)
  File "/usr/lib/python2.7/urllib2.py", line 127, in urlopen
    return _opener.open(url, data, timeout)
  File "/usr/lib/python2.7/urllib2.py", line 410, in open
    response = meth(req, response)
  File "/usr/lib/python2.7/urllib2.py", line 523, in http_response
    'http', request, response, code, msg, hdrs)
  File "/usr/lib/python2.7/urllib2.py", line 448, in error
    return self._call_chain(*args)
  File "/usr/lib/python2.7/urllib2.py", line 382, in _call_chain
    result = func(*args)
  File "/usr/lib/python2.7/urllib2.py", line 531, in http_error_default
    raise HTTPError(req.get_full_url(), code, msg, hdrs, fp)
HTTPError: HTTP Error 404: Not Found
Exception in Tkinter callback

0.11.x
```

**Comment by pelson** [comment]
> Example still fails in release 0.11.2 from Github. The URL's are still wrong.

It is rather unfortunate, but the Natural Earth guys had to remove their zips from GitHub. As a result, the URLs have changed again. There is no currently released version which points at accessible Natural Earth data (though v0.11.x does have these fixes in them thanks to #532). The workaround for non-fixed versions is to change the configuration with something like:

```
def update_config(config):
    # Ensure that the NEShpDowloader is put into the config's available
    # downloaders by importing it.
    from cartopy.io.shapereader import NEShpDownloader

    # Define the URL template.
    URL_template = 'http://naciscdn.org/naturalearth/{resolution}/{category}/ne_{resolution}_{name}.zip'

    # Get hold of the default NE downloader, and update the URL template.
    downloader = config['downloaders'][('shapefiles', 'natural_earth')]
    downloader.url_template = URL_template
```
